spec/active_force/active_query_spec.rb

```diff
@@ -40,6 +39,70 @@
     end
   end
 
+  describe '#blank? delegation' do
+    before do
+      allow(client).to receive(:query).and_return(api_result)
+    end
+
+    context 'when there are no records' do
+      let(:api_result) { [] }
+
+      it 'returns true' do
+        result = active_query.where("Text_Label = 'foo'").blank?
+        expect(result).to be true
+      end
+
+      it 'returns true' do
+        result = active_query.where("Text_Label = 'foo'").first.blank?
+        expect(result).to be true
+      end
+    end
+
+    context 'when records are returned' do
+      it 'returns false' do
+        result = active_query.where("Text_Label = 'foo'").blank?
+        expect(result).to be false
+      end
+
+      it 'returns false' do
+        result = active_query.where("Text_Label = 'foo'").first.blank?
```

Comment:
similarly here, this is testing `.first` on the ActiveQuery object to return the first Object, and then calling `.blank?` on that object. not using the delegator